vendor/src/github.com/docker/libnetwork/drivers/bridge/bridge.go

```diff
@@ -686,6 +686,15 @@ func (d *driver) EndpointOperInfo(nid, eid types.UUID) (map[string]interface{},
 
 	m := make(map[string]interface{})
 
+	if ep.config.ExposedPorts != nil {
```

Comment:
I think the way this works is that you'll need to open a PR in the libnetwork project for this change. Then once that's merged you'll need to open a PR for docker to grab the newer version of libnetwork.  We don't do "vendor" changes are part of Docker PRs - the "vendor" dir is just a copy of the remote repo at the level we want.